src/plugins/intel_gpu/include/intel_gpu/primitives/roll.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
only 2022 is needed because this file is new

Reply:
done